src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -120,110 +128,63 @@ const formConfig = {
   subTitle: 'Form 21-526EZ',
   // getHelp: GetFormHelp, // TODO: May need updated form help content
   chapters: {
-    reviewVeteranDetails: {
-      title: (isReviewPage) => {
-        const baseString = 'Review Veteran Details';
-        if (isReviewPage) {
-          return baseString.replace('Review ', '');
-        }
-        return baseString;
-      },
+    veteranDetails: {
+      title: (isReviewPage) => `${isReviewPage ? 'Review ' : ''}Veteran Details`,
       pages: {
-        veteranInformation: createVerifiedVeteranInfoPage(fullSchema526EZ),
-        specialCircumstances: { // TODO: create page file and reuse 
-          title: 'Special Circumstances',
-          path: 'review-veteran-details/special-circumstances',
-          depends: formData => formData.prefilled,
+        veteranInformation: {
+          title: 'Veteran Details', // TODO: Figure out if this is even necessary
```

Comment:
Is this, in fact, necessary?

Reply:
Is the title "Veteran Information" here?

Reply:
Oh yeah. That probably makes sense. :stuck_out_tongue_winking_eye: